src/Razor/test/Microsoft.AspNetCore.Razor.LanguageServer.Test/Completion/DefaultTagHelperServiceTestBase.cs

```diff
@@ -205,5 +228,271 @@ internal static RazorCodeDocument CreateCodeDocument(string text, string filePat
 
             return codeDocument;
         }
+
+#if GENERATE_BASELINES
+        protected bool GenerateBaselines { get; set; } = true;
+#else
+        protected bool GenerateBaselines { get; set; } = true;
```

Comment:
Should the `else` branch be defaulting `GenerateBaselines` to `true` when `#if GENERATE_BASELINES` is `false`?

Reply:
Correct, I noticed that my silly testing-edit slipped through shortly after pushing, resolved now.